src/main/java/org/ow2/proactive/catalog/service/DatabaseBackwardCompatibilityManager.java

```diff
@@ -40,10 +40,12 @@
     @Autowired
     ProjectNameStartupAdder projectNameStartupAdder;
 
+    @Autowired
+    NameKindContentTypeLowerCaseStartupAdder nameKindContentTypeLowerCaseStartupAdder;
+
     @PostConstruct
     public void initSetProjectNameInCatalogObjectRevisionEntity() {
```

Comment:
maybe rename this method to cover both synchronization

Reply:
ok